config.json

```diff
@@ -140,6 +140,17 @@
           "strings"
         ]
       },
+      {
+        "slug": "armstrong-numbers",
+        "name": "Armstrong Numbers",
+        "uuid": "1a204c41-7aea-41ff-8069-917ed65177ed",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "topics": [
+          "numbers"
+        ]
```

Comment:
You're probably aware, but for the sake of completeness: topics aren't used. You could instead have it be a prerequisite or practices.